src/services/request-service.ts

```diff
@@ -10,20 +10,30 @@ import type { RequestAnchorParams } from '../ancillary/anchor-request-params-par
 import type { IMetadataService } from './metadata-service.js'
 import type { GenesisFields } from '../models/metadata'
 import { Request, RequestStatus } from '../models/request.js'
+import { Config } from 'node-config-ts'
+import { IQueueProducerService } from './queue/queue-service.type.js'
+import { RequestQMessage } from '../models/queue-message.js'
 
 export class RequestService {
+  private readonly publishToQueue: boolean
+
   static inject = [
+    'config',
     'requestRepository',
     'requestPresentationService',
     'metadataService',
-    'anchorRequestParamsParser',
+    'validationQueueService',
   ] as const
 
   constructor(
+    config: Config,
     private readonly requestRepository: RequestRepository,
     private readonly requestPresentationService: RequestPresentationService,
-    private readonly metadataService: IMetadataService
-  ) {}
+    private readonly metadataService: IMetadataService,
+    private readonly validationQueueService: IQueueProducerService<RequestQMessage>
+  ) {
+    this.publishToQueue = config.useQueueBatches && config.queue.sqsQueueUrl !== ''
```

Comment:
Didn't we want to use just the presence or absence of the queue URL to decide v5 mode? I took `useQueueBatches` out of the config but I can add it back if preferable.

Reply:
oops missed that! will fix